### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by jnpaulson** [HCBravoLab/greengenes13.5MgDb#1]
There appears to be a bug when trying to install (at least on my mac). (See below). It comes from a potentially empty `db_dir` resulting in the `/gg_13_...`. Replacing `f <- paste0(db_dir,"/",f_basename)` with `f <- paste0(db_dir,"./",f_basename)` will fix it.

```
> require(devtools)
Loading required package: devtools
l> load_all()
Loading greengenes13.5MgDb
/gg_13_5.fasta.gz: Permission denied
Error in .Call2("new_input_ExternalFilePtr", fp, PACKAGE = "Biostrings") : 
  cannot open file ''
In addition: Warning messages:
1: In .recacheSubclasses(def@className, def, doSubclasses, env) :
  undefined subclass "externalRefMethod" of class "expressionORfunction"; definition not updated
2: In .recacheSubclasses(def@className, def, doSubclasses, env) :
  undefined subclass "externalRefMethod" of class "functionORNULL"; definition not updated
3: In download.file(db_url, destfile = f, method = "wget") :
  download had nonzero exit status
```

**Comment by jnpaulson** [comment]
However, this still leads to which I assume is an issue with the mgDB code:

```
Error in .Call2("new_input_ExternalFilePtr", fp, PACKAGE = "Biostrings") : 
  cannot open file ''
```

**Comment by hcorrada** [comment]
Fixed by d77fc478653f5f16b8dd3465c5d28db683e7f717. Can you check?

**Comment by jnpaulson** [comment]
Fixed!
